specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/applicationGateway.json

```diff
@@ -692,6 +756,65 @@
     }
   },
   "definitions": {
+    "ApplicationGatewayOnDemandProbe": {
```

Comment:
Please mark the readonly properties wherever applicable

Reply:
They are all writable in this case